SR2019/2019-03-11-competition-information.md

```diff
@@ -48,6 +48,10 @@ This number should only be used for this competition event and will only be answ
 
 With less than 4 weeks left until the final, we hope you're progressing well. The time left to get support is growing smaller, so if you're facing any issues, be sure to check the rules and the docs, they usually hold the answers! If you have any questions, please let us know on the forums, we try to answer them as quickly as possible.
 
+## Kit Loan Extension
+
+In the past, some teams have requested to hold on to the kit for a couple of weeks after the competition. If this is something you would be interested, please let us know on `teams@studentrobotics.org`, with the reason you wish to hold on to the kit. The closing date for this is 31st March.
```

Comment:
Ahem, the rules spec this as 22nd March. We should be consistent (but I don't mind which date we use).